src/test/java/gil/server/HandlerTest.java

```diff
@@ -4,16 +4,22 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.StringReader;
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
 
 public class HandlerTest {
     Handler handler = new Handler();
 
     @Test
     public void shouldReturnAResponseObject() throws IOException {
         String data = "GET / HTTP/1.1 text/html\r\n";
+        String expectedBody = "Hello, world!";
+        String expectedStatusCode = "200";
         StringReader dataStream = new StringReader(data);
         BufferedReader bufferedReader = new BufferedReader(dataStream);
-        assertTrue (handler.processRequest(bufferedReader) instanceof Response);
+
+        Response result = handler.processRequest(bufferedReader);
+
+        assertEquals(expectedBody, result.getBody());
+        assertEquals(expectedStatusCode, result.getStatusCode());
     }
-}
+}
```

Comment:
Many editors have an option to automatically insert a newline at the end of each file on save.